sdk/examples/client/block/00_block_no_payload.rs

```diff
@@ -10,8 +10,9 @@ use iota_sdk::client::{Client, Result};
 async fn main() -> Result<()> {
     // Take the node URL from command line argument or use one from env as default.
     let node_url = std::env::args().nth(1).unwrap_or_else(|| {
-        // This example uses dotenv, which is not safe for use in production.
-        dotenv::dotenv().ok();
+        // This example uses secrets in environment variables for simplicity but should generally not be done in
```

Comment:
Would it be one line if we remove `generally`? Not sure why we would say that anyway. It opens up doubt, we should tell them to ever do that.